src/standard-live-connect.ts

```diff
@@ -119,23 +119,23 @@ function _standardInitialization (liveConnectConfig: LiveConnectConfig, external
   }
 }
 
-function _initializeWithoutGlobalName(liveConnectConfig: LiveConnectConfig, externalStorageHandler: ExternalStorageHandler, externalCallHandler: ExternalCallHandler, eventBus: EventBus): ILiveConnect {
+function _initializeWithoutGlobalName(liveConnectConfig: LiveConnectConfig, externalStorageHandler: StorageHandler, externalCallHandler: ExternalCallHandler, eventBus: EventBus): ILiveConnect {
```

Comment:
ExternalCallHandler? WTF?

Reply:
Good questions.

Reply:
Good catch, consider to change it ExternalCallHandler -> CallHandler like you did in some other places.